src/ophys_etl/transforms/mesoscope_2p/tiff.py

```diff
@@ -262,11 +264,21 @@ def volume_scans(self):
 
```

Comment:
@nataliaorlova pointed out to me that there is a `channelSave` parameter in the metadata that I wasn't aware of, which seems to actually determine which channels are saved to file instead of `channelsActive`. So that is a fix for someone who is able to do more testing on the mesoscope rig.